pkg/raft/BUILD.bazel

```diff
@@ -39,7 +39,6 @@ go_test(
         "interaction_test.go",
         "log_test.go",
         "log_unstable_test.go",
-        "node_bench_test.go",
```

Comment:
This line probably needs to stay. There is a second file with the same name, which is not deleted here?

Reply:
tip: use `dev gen bazel` to get this right